docs/blog/introducing-check-html-links.md

```diff
@@ -0,0 +1,166 @@
+So we know that theoretically, it should be possible to be fast - but what should be our goal?
+To not disturb too much during writing/development it should be finished within 1s for a small site (up to 200 pages).
+For medium sites (2000-1000 pages), it's reasonable if it takes a little longer - let's aim for less than 5 seconds. This will probably be a breaking point where you execute it only on-demand and in the CI instead of executing it on every save.
+
+## Requirement Node Based & Small Dependency Tree
+
+The end goal is to integrate it within a bigger WIP system called Rocket which is node-based so therefore it will need to at least support node. Having it standalone makes it more versatile and easier to maintain/test.
+
+Last but not least the JavaScript ecosystem is a "violent" environment. Lot's of changes/improvements happen all the time. It's often hard to keep up. Therefore having a small dependency tree is something to always thrive for.
+
+## Solution
+
+In the end because of the requirements I had to implement yet another solution 😅
+
+I call it `check-html-links` - e.g. no broken links or assets
+
+The Features are:
+
+- extracts all href, src, srset
+- use a wasm parser (sax-wasm)
+- stream the html for performance
+- check if file or id within file exist
+- focus on missing references/sources
+
+## Usage
+
+It does check your final html output so you need to execute it after your Static Site Generator.
+
+```
+npx check-html-links _site
+```
+
+## Usage Github Action
+
+There is a Github action available for it. You can find it in the marketplace at https://github.com/marketplace/actions/check-html-links-action.
+
+Here is a complete Example workflow that will check the result of the folder `_site` in the root of your repository on each push:
+
+```yml
+on: [push]
+
+jobs:
+  check_html_links_job:
+    runs-on: ubuntu-latest
+    name: A job to test check-html-links-action
+    steps:
+      - uses: actions/checkout@v2
+      - name: check-html-links-action step
+        id: check-links
+        uses: modernweb-dev/check-html-links-action@v1
+        with:
+          doc-folder: '_site_'
+```
+
+## Comparison
+
+Checking the output of [11ty-website](https://github.com/11ty/11ty-website) with 13 missing reference targets (used by 516 links) while checking 501 files. (on January 17, 2021)
+
+| Tool             | Lines printed | Duration | Lang | Dependency Tree |
+| ---------------- | ------------- | -------- | ---- | --------------- |
+| check-html-links | 38            | ~2.5s    | node | 19              |
+| link-checker     | 3000+         | ~11s     | node | 106             |
+| hyperlink        | 68            | 4m 20s   | node | 481             |
+| htmltest         | 1000+         | ~0.7s    | GO   | -               |
+
+
+## Thanks
+
+Thank you for following along on my journey on creating `check-html-links`. You can find the code on [Github](https://github.com/modernweb-dev/rocket/tree/main/packages/check-html-links). 
```

Comment:
I'd probably add something about what you'd like to add to the tool or whether it is finished in your opinion

Reply:
added a section above - thx for that tip 👍